assets/locales/service.cy.toml

```
@@ -0,0 +1,66 @@
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+# Notes
+# - Documentation: https://github.com/nicksnyder/go-i18n/blob/master/README.md
+# - Description should be the English version of the text, unless English version is a shorthand e.g. Home button is shorthand in English for Homepage
+# - Arguments can be added via {{.argN}} where N is integer index starting at 0 e.g. {{.arg0}} {{.arg1}}
+# - To inject HTML (Avoid where possible as it is messy) use "| safeHTML" e.g. {{ localise Foo .Language 1 | safeHTML }} where Foo might be "<strong>test</strong>"
+# count - 0 = zero
+# count - 1 = one
+# count - 2 = two; useful for languages with mutations like welsh (radical) 'brother'=brawd, two (soft) 'brothers'=frawd, three or more (nasal) 'brothers'=brodyr
+# count - 3 = few
+# count - 6 = many
+# count - 4 = other; 4,5 & >6
+# Plural rules can be found here: http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+[FindFactsAndFiguresEnglandWales]
+description = "Find facts and figures heading"
+one = "Find facts and figures about areas in England or Wales"
+
+[GetDataAboutPeopleAndHouseholds]
+description = "Area profiles start page intro"
+one = "Get data about people and households in <a href='/areas/E92000001'>England</a> or <a href='/areas/W92000004'>Wales</a>. You can also view facts and figures for a town, local authority or ward in each country."
```

Use double quote for attribute values. You can escape them 
```suggestion
one = "Get data about people and households in <a href=\"/areas/E92000001\">England</a> or <a href=\"/areas/W92000004\">Wales</a>. You can also view facts and figures for a town, local authority or ward in each country."
```